AUTHORS.txt

```diff
@@ -34,4 +34,6 @@ Yutaka Takeda <yt0916@gmail.com>
 ZHENK <chengzhenyang@gmail.com>
 
 # List of contributors not appearing in Git history
-
+The gortc project
+The IETF Trust
+Aliaksandr Valialkin <valyala@gmail.com>
```

Comment:
Generate-authors script will sort this block but not automatically pushed since this PR is from the fork.

Reply:
Thanks :) I fixed it in the latest version.